src/custom-addons/orr_document_extend/models/ir_attachment.py

```diff
@@ -0,0 +1,21 @@
+from odoo import models, fields,api,_
+from odoo.exceptions import UserError
```

Comment:
removing unused imports
```suggestion
from odoo import models
```